08-projects.Rmd

```diff
@@ -6168,34 +6195,34 @@ If you paste a `title_id` to the end of the following url, it will pull up the p
 - Output of the query.
 ```
 
-##### 4. The `episode_title_id` column in the `episodes` table references titles of individual episodes of a tv series. The `show_title_id` references the titles of the show itself. With that in mind, write a query that gets a list of all of the episodes and titles of Grey's Anatomy.
+##### 4. The `episode_title_id` column in the `episodes` table references titles of individual episodes of a TV series. The `show_title_id` references the titles of the show itself. With that in mind, write a query that gets a list of all of the episodes and their titles of _Grey's Anatomy_.
```

Comment:
Hi @kevinamstutz , I added "their" with the intention to emphasize that the students should retrieve information also from the `titles` table. My reason was it seemed a bit unclear what was asked when I read it the first time: Although the "relevant topics" gives INNER JOIN, still ended up only writing a query based on `episodes`. But I am not sure whether adding "their" would work. What do you think?

Reply:
Actually, this question is essentially the same as Question 6 (finding episodes of one's favorite show). Question 6 is straight-forward in that the relevant tables are directly given in the question description. Do we want to do the same here for Question 4?

Reply:
I've modified questions 4 & 6 accordingly to make this more straightforward and clear, let me know what you think (when I push it)

Reply:
Hi @kevinamstutz , I think the updated Questions 4 and 6 look nice!